kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/util/Metrics.java

```diff
@@ -280,4 +283,17 @@ public interface ClusterNodeSpecificMetricProviderCreator<T extends Meter> {
 
         MeterProvider<T> create(String virtualCluster, Integer nodeId);
     }
-}
\ No newline at end of file
+
+    public static void versionInfoMetric(VersionInfo versionInfo) {
+        // Something is wrong here
+        // https://github.com/micrometer-metrics/micrometer/wiki/1.13-Migration-Guide#info-vs-gauge-type
+        // says that suffixing .info should cause mircometer to use Prometheus's info type.
+        // but that's no what I'm seeing.
```

Comment:
The metrics getting the right name (`kroxylicious_build_info`) by the type is Gauge rather than Info.

Reply:
So what do you propose we do for the next release?

Reply:
I don't think it is impactful, but I need to confirm that.  I guess the worst that can happen is the PromQL editor offers some inappropriate completions.    There might be a defect to raise against micrometer.

Reply:
:+1:

Reply:
There's actually no issue here.  "gauge" is the expected type string in this case. [ExpositionFormatWriter](https://prometheus.io/docs/instrumenting/exposition_formats/#comments-help-text-and-type-information)